contrib/win32/win32compat/w32-sshfileperm.c

```diff
@@ -170,10 +170,8 @@ is_sshd_account(PSID user_sid) {
 		debug3("LookupAccountSidLocalW() failed with error: %d. ", GetLastError());
 		errno = ENOENT;
 		return FALSE;
-	}
-	domain_name_length = wcslen(full_name);
-	full_name[domain_name_length] = L'\\';
```

Comment:
What issue was this causing (why was it replaced with wcsncat_s that is less performant)?